clang/test/CodeGen/aie/aie2/acc32.cpp

```diff
@@ -8,7 +8,7 @@
 // (c) Copyright 2023-2024 Advanced Micro Devices, Inc. or its affiliates
 //
 //===----------------------------------------------------------------------===//
-// RUN: %clang --target=aie2 -S -emit-llvm %s -o - | FileCheck %s
+// RUN: %clang --target=aie2 -nostdlibinc -S -emit-llvm %s -o - | FileCheck %s
```

Comment:
I'm a bit concerned at having to sprinkle this everywhere.  Is this just because these tests might need to run without the runtimes having been built?

Reply:
Yes this is needed because we call the `%clang` driver here instead of `-cc1`. That expects the runtimes to be present, which we cannot rely on in clang codegen tests.
Applications will not have to set this option